datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
or may be that would be counter-productive since may be the reason for failure is the same what is actually triggering other tests to fail and functionality for those need to be fixed up first???
```shell
$> wget -O- -q https://s3.amazonaws.com/archive.travis-ci.org/jobs/159374278/log.txt | grep FAIL: | grep -v crawler
FAIL: datalad.interface.tests.test_save.test_recursive_save
FAIL: datalad.interface.tests.test_save.test_save
FAIL: datalad.interface.tests.test_utils.test_dirty
```